recipe/meta.yaml

```diff
@@ -1,229 +1,317 @@
+        - cfitsio
+        - curl
+        - expat
+        - freexl
+        - geos
+        - geotiff
+        - giflib   # [not win]
+        - hdf4
+        - hdf5
+        - icu
+        - jpeg
+        - json-c   # [not win]
+        - kealib
+        - arrow-cpp
+        - libkml
+        - libnetcdf
+        - libpng
+        - libpq
+        - libspatialite
+        - libtiff
+        - libuuid  # [linux]
+        - libwebp-base
+        - libxml2
+        - openjpeg
+        - openssl
+        - pcre2
+        - poppler
+        - postgresql
+        - proj
+        - qhull
+        - sqlite
+        - tiledb
+        - xerces-c
+        - xz
+        - zlib
+        - zstd
+
+      run:
+        - {{ pin_subpackage('libgdal', max_pin='x.x.x', exact=True) }}
+        - cfitsio
+        - geotiff
+        - giflib   # [not win]
+        - json-c   # [not win]
+        - libpq
+        - libspatialite
+        - libuuid  # [linux]
+        - libwebp-base
+        - poppler
+        - postgresql
+        - proj
+        - tiledb
+        - xz
+        - zstd
+
+    test:
+      files:
+        - test_arrow_parquet.sh   # [unix]
+        - test_arrow_parquet.bat  # [win]
       commands:
-        - gdal_grid --version
-        - gdal_rasterize --version
-        - gdal_translate --version
-        - gdaladdo --version
-        - gdalenhance --version
-        - gdalwarp --version
-        - gdalinfo --formats
-        # (tkoch) this one fails often, maybe because of connectivity issues.
-        #- gdalinfo http://thredds.nersc.no/thredds/dodsC/greenpath/Model/topaz  # [not win]
-        - {{ RECIPE_DIR }}/run_test.bat  # [win]
-        - bash {{ RECIPE_DIR }}/run_test.sh  # [not win]
+        - sh test_arrow_parquet.sh   # [unix]
+        - test_arrow_parquet.bat  # [win]
+
     about:
-      home: https://www.gdal.org
+      summary: Arrow and Parquet drivers for the Geospatial Data Abstraction Library (GDAL)
       license: MIT
       license_family: MIT
       license_file: LICENSE.TXT
-      summary: The Geospatial Data Abstraction Library (GDAL)
-      description: |
-        GDAL is a translator library for raster and vector geospatial data formats that is released under an
-        X/MIT style Open Source license by the Open Source Geospatial Foundation.
-      dev_url: https://github.com/OSGeo/gdal
-      doc_url: https://gdal.org/tutorials/index.html
 
   - name: gdal
-    script: install_python.sh   # [unix]
+    script: install_python.sh  # [unix]
     script: install_python.bat  # [win]
     build:
-      ignore_run_exports:
-        - openssl
-        - hdf5
+      skip_compile_pyc:
+        - share/bash-completion/completions/*.py
+
     requirements:
       build:
         - {{ compiler('c') }}
         - {{ compiler('cxx') }}
-        - cmake  # [win]
-        - make   # [not win]
+        - cmake
+        - make   # [unix]
+        - ninja  # [win]
+
       host:
         - python
-        - setuptools <58.0.0
```

Comment:
Missing setuptools and wheel

Reply:
add those dependencies. thanks for noticing